### DISCUSSION TITLE
Remove Core column have max handle the rest

### DISCUSSION THREAD

**Discussion by MWhal** [dmdorman/hero6e-foundryvtt#698] [Ideas]
I'm still looking into this, but what is everyone's opinion if I were to remove the **CORE** value, and let **MAX** start take the heavy lifting. MAX will then be the thing the HDC file updates first, and we would be one step closer to having HDC lite. 
Note* this is just a thought that crossed my mind while at work; most likely will experiment with it later.

**Comment by aeauseth** [comment]
So the CORE value is used to determine the cost of characteristics.  You could probably remove the display of the column and possibly BASE and COST if you really wanted to.  A mouse over hover could be used to provide any information for columns you hide/remove.  The original intent was to closely match Hero Designer.  I added CORE for debugging and never considered removing it.

Under the hood I think we need CORE (willing to entertain a renaming of this variable) to properly calculate the cost of characteristics.

Also if you remove CORE, how are you going to let players "buy" characteristics with character points?  The whole in gamed editing (HD lite).
